### ISSUE TITLE
источник luftdaten.info

### ISSUE THREAD

**Issue by AlexParshin79** [kuzin2006/hass_save_eco_bot_sensor#3]
Привет! 
отличный сенсор, спасибо!
в последних версиях HASS требует версию в манифесте, я добавил и запустил сенсор
я так понимаю источник данных для сенсора это SAVEDNIPRO
добавь пожалуйста еще источник luftdaten.info 
https://www.saveecobot.com/platform/luftdaten

**Comment by D1monUA** [comment]
> Привет! отличный сенсор, спасибо! в последних версиях HASS требует версию в манифесте, я добавил и запустил сенсор я так понимаю источник данных для сенсора это SAVEDNIPRO добавь пожалуйста еще источник luftdaten.info https://www.saveecobot.com/platform/luftdaten

Подскажите, как вы запустили сенсор?
У меня при проверке конфига ошибка - Platform error sensor.save_eco_bot - Integration 'save_eco_bot' not found.
Попробовал в файле манифеста добавить версию, но без резуьтатов

**Comment by AlexParshin79** [comment]
> > Привет! отличный сенсор, спасибо! в последних версиях HASS требует версию в манифесте, я добавил и запустил сенсор я так понимаю источник данных для сенсора это SAVEDNIPRO добавь пожалуйста еще источник luftdaten.info https://www.saveecobot.com/platform/luftdaten
> 
> Подскажите, как вы запустили сенсор? У меня при проверке конфига ошибка - Platform error sensor.save_eco_bot - Integration 'save_eco_bot' not found. Попробовал в файле манифеста добавить версию, но без резуьтатов

**Comment by AlexParshin79** [comment]
по пути /config/custom_components/save_eco_bot/ закинули файлы ?

**Comment by D1monUA** [comment]
> по пути /config/custom_components/save_eco_bot/ закинули файлы ?

Да, закинул, перезагрузился, ошибка эта пропала, но интеграция не стартонула, в логах выскочила ошибка, про которую написал сюда https://github.com/kuzin2006/hass_save_eco_bot_sensor/issues/4

**Comment by VahaC** [comment]
https://github.com/andr7000/hass_save_eco_bot_sensor
